docs/pages/architecture/proxy-peering.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Proxy Peering
+description: How Teleport implements more efficient networking with proxy peering.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  Proxy peering is available starting from Teleport `10.0`.
+</Details>
+
+## Overview
```

Comment:
I think we can remove this, since it's implied that the first paragraph is an overview.